### ISSUE TITLE
Log spam related to armor items

### ISSUE THREAD

**Issue by Birbolion** [SlimeKnights/TinkersConstruct#5065]
### Minecraft Version

1.18.2

### Forge Version

40.0.2

### Mantle Version

1.9.43

### Tinkers' Construct Version

9.1.0.11

### Describe your issue

~~This has to stem from the current plate armor embellishment issue,~~ but seems to be interacting strangely with other mods when stack tracing, despite most of these mods not interacting with Tinkers at all to begin with. This is not a report on an isolated issue, I am simply just trying to help out in current endeavors.
I have isolated one of these traces for reference.

[19Jan2023 21:09:54.988] [Worker-Main-3/WARN] [tconstruct/]: Tool stack constructed using non-modifiable tool, this may cause issues as it has no NBT. Stacktrace can be disabled in config.
java.lang.Exception: Stack trace
	at slimeknights.tconstruct.library.tools.nbt.ToolStack.from(ToolStack.java:145) ~[TConstruct-1.18.2-3.6.1.88.jar%23171!/:1.18.2-3.6.1.88]
	at slimeknights.tconstruct.library.tools.nbt.ToolStack.from(ToolStack.java:163) ~[TConstruct-1.18.2-3.6.1.88.jar%23171!/:1.18.2-3.6.1.88]
	at slimeknights.tconstruct.library.tools.item.ModifiableArmorItem.setDamage(ModifiableArmorItem.java:209) ~[TConstruct-1.18.2-3.6.1.88.jar%23171!/:1.18.2-3.6.1.88]
	at net.minecraft.world.item.ItemStack.m_41721_(ItemStack.java:300) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.world.item.ItemStack.<init>(ItemStack.java:147) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.world.item.ItemStack.<init>(ItemStack.java:138) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.world.item.ItemStack.<init>(ItemStack.java:126) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.advancements.DisplayInfo.m_14986_(DisplayInfo.java:113) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.advancements.DisplayInfo.m_14981_(DisplayInfo.java:96) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.advancements.Advancement$Builder.fromJson(Advancement.java:332) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.ServerAdvancementManager.m_136037_(ServerAdvancementManager.java:48) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at java.util.HashMap.forEach(Unknown Source) ~[?:?]
	at net.minecraft.server.ServerAdvancementManager.m_5787_(ServerAdvancementManager.java:45) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.ServerAdvancementManager.m_5787_(ServerAdvancementManager.java:24) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_10834_(SimpleReloadInstance.java:68) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_10834_(SimpleReloadInstance.java:68) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_10834_(SimpleReloadInstance.java:68) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_10834_(SimpleReloadInstance.java:68) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_10834_(SimpleReloadInstance.java:68) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_10834_(SimpleReloadInstance.java:68) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_143940_(SimpleReloadInstance.java:69) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance.m_10834_(SimpleReloadInstance.java:68) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at java.util.concurrent.CompletableFuture$UniCompletion.claim(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(Unknown Source) ~[?:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance$1.m_10855_(SimpleReloadInstance.java:51) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at net.minecraft.server.packs.resources.SimpleReloadInstance$1.m_6769_(SimpleReloadInstance.java:48) ~[server-1.18.2-20220404.173914-srg.jar%23206!/:?]
	at java.util.concurrent.CompletableFuture$UniCompose.tryFire(Unknown Source) [?:?]
	at java.util.concurrent.CompletableFuture.postComplete(Unknown Source) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinTask.doExec(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinPool.scan(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source) [?:?]

~~Other than that, the same visual bug with the plate armor not rendering textures appears. Hopefully this will help out in solving the issue.~~

### Other mods

[Waystones (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/waystones)
[Architectury API (Fabric/Forge) (by shedaniel)](https://www.curseforge.com/minecraft/mc-mods/architectury-api)
[Shulker Box Slot (Forge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/shulker-box-slot)
[Experience Obelisk (by arcatex177)](https://www.curseforge.com/minecraft/mc-mods/experience-obelisk)
[Polymorph (Forge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/polymorph)
[Simple Storage Network (by Lothrazar)](https://www.curseforge.com/minecraft/mc-mods/simple-storage-network)
[HT's TreePlant (by hammertater)](https://www.curseforge.com/minecraft/mc-mods/treeplant)
[YUNG's Extras (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-extras)
[Farmer's Respite (by lumpazl)](https://www.curseforge.com/minecraft/mc-mods/farmers-respite)
[Enchantment Descriptions (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/enchantment-descriptions)
[JEI Integration (by SnowShock35)](https://www.curseforge.com/minecraft/mc-mods/jei-integration)
[Create Stuff & Additions (by Furti_Two)](https://www.curseforge.com/minecraft/mc-mods/create-stuff-additions)
[Tinkers Construct (by mDiyo)](https://www.curseforge.com/minecraft/mc-mods/tinkers-construct)
[Snad (by TheRoBrit)](https://www.curseforge.com/minecraft/mc-mods/snad)
[Large Meals (An Add-on for Farmer's Delight) (by LostWiz08)](https://www.curseforge.com/minecraft/mc-mods/large-meals-an-add-on-for-farmers-delight)
[Chisels & Bits - For Forge (by AlgorithmX2)](https://www.curseforge.com/minecraft/mc-mods/chisels-bits)
[Dungeon Crawl (by xiroc_)](https://www.curseforge.com/minecraft/mc-mods/dungeon-crawl)
[Easy Villagers (by henkelmax)](https://www.curseforge.com/minecraft/mc-mods/easy-villagers)
[Emojiful (by Buuz135)](https://www.curseforge.com/minecraft/mc-mods/emojiful)
[Infernal Expansion (by InfernalStudios)](https://www.curseforge.com/minecraft/mc-mods/infernal-expansion)
[Art of Forging: A Tetra Addon (by Eldritch_King24)](https://www.curseforge.com/minecraft/mc-mods/art-of-forging-a-tetra-addon)
[Easy Magic (by Fuzs_)](https://www.curseforge.com/minecraft/mc-mods/easy-magic)
[Magnesium/Rubidium Extras (by TeamDeusVult)](https://www.curseforge.com/minecraft/mc-mods/magnesium-extras)
[Dual Riders (by Flytre7)](https://www.curseforge.com/minecraft/mc-mods/dual-riders)
[Valhelsia Structures (by ValhelsiaTeam)](https://www.curseforge.com/minecraft/mc-mods/valhelsia-structures)
[Placebo (by Shadows_of_Fire)](https://www.curseforge.com/minecraft/mc-mods/placebo)
[Guns Without Roses (by Lykrast)](https://www.curseforge.com/minecraft/mc-mods/guns-without-roses)
[Tool Belt (by gigaherz)](https://www.curseforge.com/minecraft/mc-mods/tool-belt)
[mutil (by mickelus)](https://www.curseforge.com/minecraft/mc-mods/mutil)
[CoFH Core (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/cofh-core)
[Death Compass (by vadis365)](https://www.curseforge.com/minecraft/mc-mods/death-compass)
[Presence Footsteps [Forge] (by Paint_Ninja)](https://www.curseforge.com/minecraft/mc-mods/presence-footsteps-forge)
[Cosmetic Armor Reworked (by LainMI)](https://www.curseforge.com/minecraft/mc-mods/cosmetic-armor-reworked)
[Comforts (Fabric/Forge/Quilt) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/comforts)
[♪ Alex's Mobs EXTRA Music ♪ (by theexiiedfeiiow_)](https://www.curseforge.com/minecraft/mc-mods/alexs-mobs-battle-music)
[Collective (by Serilum)](https://www.curseforge.com/minecraft/mc-mods/collective)
[Flywheel (by jozufozu)](https://www.curseforge.com/minecraft/mc-mods/flywheel)
[Immersive Aircraft [Fabric/Forge] (by Conczin)](https://www.curseforge.com/minecraft/mc-mods/immersive-aircraft)
[Clumps (by Jaredlll08)](https://www.curseforge.com/minecraft/mc-mods/clumps)
[Immersive Armors [Fabric/Forge] (by Conczin)](https://www.curseforge.com/minecraft/mc-mods/immersive-armors)
[SuperMartijn642's Core Lib (by SuperMartijn642)](https://www.curseforge.com/minecraft/mc-mods/supermartijn642s-core-lib)
[Locksmith (by moonflowerteam)](https://www.curseforge.com/minecraft/mc-mods/locksmith)
[More Dragon Eggs (by Darkere)](https://www.curseforge.com/minecraft/mc-mods/more-dragon-eggs)
[Chipped (by terrariumearth)](https://www.curseforge.com/minecraft/mc-mods/chipped)
[AutoRegLib (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/autoreglib)
[FindMe (by Buuz135)](https://www.curseforge.com/minecraft/mc-mods/findme)
[Cooking for Blockheads (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/cooking-for-blockheads)
[Croptopia (by thethonk)](https://www.curseforge.com/minecraft/mc-mods/croptopia)
[GraveStone Mod (by henkelmax)](https://www.curseforge.com/minecraft/mc-mods/gravestone-mod)
[Just Enough Resources (JER) (by way2muchnoise)](https://www.curseforge.com/minecraft/mc-mods/just-enough-resources-jer)
[Iron Chests (by ProgWML6)](https://www.curseforge.com/minecraft/mc-mods/iron-chests)
[Xtra Arrows (by Jackbusters1)](https://www.curseforge.com/minecraft/mc-mods/xtra-arrows)
[Easy Piglins (by henkelmax)](https://www.curseforge.com/minecraft/mc-mods/easy-piglins)
[Mantle (by mDiyo)](https://www.curseforge.com/minecraft/mc-mods/mantle)
[Thermal Expansion (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-expansion)
[TheObsidianBoat (by UnRealDinnerbone)](https://www.curseforge.com/minecraft/mc-mods/theobsidianboat)
[Puzzles Lib (by Fuzs_)](https://www.curseforge.com/minecraft/mc-mods/puzzles-lib)
[Cucumber Library (by BlakeBr0)](https://www.curseforge.com/minecraft/mc-mods/cucumber)
[Thermal Cultivation (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-cultivation)
[Alex's Mobs (by sbom_xela)](https://www.curseforge.com/minecraft/mc-mods/alexs-mobs)
[MrCrayfish's Furniture Mod (by MrCrayfish)](https://www.curseforge.com/minecraft/mc-mods/mrcrayfish-furniture-mod)
[FastFurnace (by Shadows_of_Fire)](https://www.curseforge.com/minecraft/mc-mods/fastfurnace)
[Too Fast (by Noobanidus)](https://www.curseforge.com/minecraft/mc-mods/too-fast)
[ForgeEndertech (by EnderLanky)](https://www.curseforge.com/minecraft/mc-mods/forgeendertech)
[Tetra (by mickelus)](https://www.curseforge.com/minecraft/mc-mods/tetra)
[Mouse Tweaks (by YaLTeR)](https://www.curseforge.com/minecraft/mc-mods/mouse-tweaks)
[Tetra Pak (by Noobulus_Dos)](https://www.curseforge.com/minecraft/mc-mods/tetra-pak)
[Citadel (by sbom_xela)](https://www.curseforge.com/minecraft/mc-mods/citadel)
[Just Enough Items (JEI) (by mezz)](https://www.curseforge.com/minecraft/mc-mods/jei)
[Ars Nouveau (by baileyholl2)](https://www.curseforge.com/minecraft/mc-mods/ars-nouveau)
[MmmMmmMmmMmm (Target Dummy) (by MehVahdJukaar)](https://www.curseforge.com/minecraft/mc-mods/mmmmmmmmmmmm)
[Farsight [Forge] (by someaddon)](https://www.curseforge.com/minecraft/mc-mods/farsight)
[Storage Drawers (by Texelsaur)](https://www.curseforge.com/minecraft/mc-mods/storage-drawers)
[Grappling Hook Mod (by yyonne)](https://www.curseforge.com/minecraft/mc-mods/grappling-hook-mod)
[Customized Dungeon Loot (by XxRexRaptorxX)](https://www.curseforge.com/minecraft/mc-mods/customized-dungeon-loot)
[Spirit (by CodexAdrian)](https://www.curseforge.com/minecraft/mc-mods/spirit)
[Builder's Delight (Forge) (by Tynoxs)](https://www.curseforge.com/minecraft/mc-mods/builders-delight-forge)
[Thermal Locomotion (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-locomotion)
[Materialis (by RCXcrafter)](https://www.curseforge.com/minecraft/mc-mods/materialis)
[Mob Grinding Utils (by vadis365)](https://www.curseforge.com/minecraft/mc-mods/mob-grinding-utils)
[Advanced Chimneys (by EnderLanky)](https://www.curseforge.com/minecraft/mc-mods/advanced-chimneys)
[Aquatic Torches (by realmayus)](https://www.curseforge.com/minecraft/mc-mods/aquatic-torches)
[Better Compatibility Checker (by Gaz_)](https://www.curseforge.com/minecraft/mc-mods/better-compatibility-checker)
[Enlightened End (Remade) (by MuffinsQw)](https://www.curseforge.com/minecraft/mc-mods/enlightened-end)
[YUNG's Better Desert Temples (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-desert-temples)
[YUNG's Better Strongholds (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-strongholds)
[Entity Culling Fabric/Forge (by tr9zw)](https://www.curseforge.com/minecraft/mc-mods/entityculling)
[Simple Discord RPC [Forge/Fabric/Quilt] (by hypherionsa)](https://www.curseforge.com/minecraft/mc-mods/simple-discord-rpc)
[YUNG's Bridges (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-bridges)
[Caelus API (Forge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/caelus)
[Additional Structures (by XxRexRaptorxX)](https://www.curseforge.com/minecraft/mc-mods/additional-structures)
[Thermal Foundation (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-foundation)
[Nether's Delight (by lumpazl)](https://www.curseforge.com/minecraft/mc-mods/nethers-delight)
[Lootr (Forge) (by Noobanidus)](https://www.curseforge.com/minecraft/mc-mods/lootr)
[Grimoire of Tetra (by Vox4l)](https://www.curseforge.com/minecraft/mc-mods/grimoire-of-tetra)
[Enchantment Transfer [Forge] (by treekicker4)](https://www.curseforge.com/minecraft/mc-mods/enchantment-transfer-forge)
[Portality (by Buuz135)](https://www.curseforge.com/minecraft/mc-mods/portality)
[Immersive Engineering (by BluSunrize)](https://www.curseforge.com/minecraft/mc-mods/immersive-engineering)
[Create (by simibubi)](https://www.curseforge.com/minecraft/mc-mods/create)
[Jade Addons (Forge) (by Snownee_)](https://www.curseforge.com/minecraft/mc-mods/jade-addons)
[Create Deco (by talrey)](https://www.curseforge.com/minecraft/mc-mods/create-deco)
[Ars Elemental (by Alexth99)](https://www.curseforge.com/minecraft/mc-mods/ars-elemental)
[Packing Tape (by gigaherz)](https://www.curseforge.com/minecraft/mc-mods/packing-tape)
[Trash Cans (by SuperMartijn642)](https://www.curseforge.com/minecraft/mc-mods/trash-cans)
[Mekanism (by bradyaidanc)](https://www.curseforge.com/minecraft/mc-mods/mekanism)
[Jade 🔍 (by Snownee_)](https://www.curseforge.com/minecraft/mc-mods/jade)
[YUNG's API (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-api)
[Cloth Config API (Fabric/Forge) (by shedaniel)](https://www.curseforge.com/minecraft/mc-mods/cloth-config)
[EnderTanks (by ShetiPhian)](https://www.curseforge.com/minecraft/mc-mods/endertanks)
[Controlling (by Jaredlll08)](https://www.curseforge.com/minecraft/mc-mods/controlling)
[Bookshelf (by DarkhaxDev)](https://www.curseforge.com/minecraft/mc-mods/bookshelf)
[YUNG's Better Witch Huts (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-witch-huts)
[JEI Enchantment Info (by Phylogeny)](https://www.curseforge.com/minecraft/mc-mods/jei-enchantment-info)
[ShetiPhianCore (by ShetiPhian)](https://www.curseforge.com/minecraft/mc-mods/shetiphiancore)
[YUNG's Better Mineshafts (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-mineshafts-forge)
[Corail Recycler (by Corail_31)](https://www.curseforge.com/minecraft/mc-mods/corail-recycler)
[Simple Shops (by wolforce)](https://www.curseforge.com/minecraft/mc-mods/simple-shops)
[Iceberg [Forge] (by Grend_G)](https://www.curseforge.com/minecraft/mc-mods/iceberg)
[Industrial Foregoing (by Buuz135)](https://www.curseforge.com/minecraft/mc-mods/industrial-foregoing)
[Login Shield (by Glasspelican)](https://www.curseforge.com/minecraft/mc-mods/login-shield)
[The Undergarden (by Quek04)](https://www.curseforge.com/minecraft/mc-mods/the-undergarden)
[Curios API (Forge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/curios)
[Create Crafts & Additions (by MRHminer)](https://www.curseforge.com/minecraft/mc-mods/createaddition)
[Tinkers' Rapier (by sz0999312)](https://www.curseforge.com/minecraft/mc-mods/tinkers-rapier)
[Farmer's Delight (by vectorwing)](https://www.curseforge.com/minecraft/mc-mods/farmers-delight)
[YUNG's Better Dungeons (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/yungs-better-dungeons)
[Double Slabs (by CJMinecraft01)](https://www.curseforge.com/minecraft/mc-mods/double-slabs)
[Titanium (by Buuz135)](https://www.curseforge.com/minecraft/mc-mods/titanium)
[The Twilight Forest (by Benimatic)](https://www.curseforge.com/minecraft/mc-mods/the-twilight-forest)
[L_Ender 's Cataclysm (by mcl_ender)](https://www.curseforge.com/minecraft/mc-mods/l_ender-s-cataclysm)
[Flytre Lib (FlytreLib) (by Flytre7)](https://www.curseforge.com/minecraft/mc-mods/lib)
[Thermal Integration (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-integration)
[Traveler's Titles (Forge) (by YUNGNICKYOUNG)](https://www.curseforge.com/minecraft/mc-mods/travelers-titles)
[Blood Magic (by WayofTime)](https://www.curseforge.com/minecraft/mc-mods/blood-magic)
[Pollen (by moonflowerteam)](https://www.curseforge.com/minecraft/mc-mods/pollen)
[Carry On (by Tschipp)](https://www.curseforge.com/minecraft/mc-mods/carry-on)
[Better Third Person (by Socolio)](https://www.curseforge.com/minecraft/mc-mods/better-third-person)
[Golden Hopper (by MrCrayfish)](https://www.curseforge.com/minecraft/mc-mods/golden-hopper)
[Thermal Innovation (by TeamCoFH)](https://www.curseforge.com/minecraft/mc-mods/thermal-innovation)
[Charm of Undying (Fabric/Forge/Quilt) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/charm-of-undying)
[Xaero's World Map (by xaero96)](https://www.curseforge.com/minecraft/mc-mods/xaeros-world-map)
[Farming for Blockheads (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/farming-for-blockheads)
[More Ore Stones [FORGE] (by OrdanaryMods)](https://www.curseforge.com/minecraft/mc-mods/mores-forge)
[Friends&Foes (Forge) (by faboslav)](https://www.curseforge.com/minecraft/mc-mods/friends-and-foes-forge)
[Mekanism Generators (by bradyaidanc)](https://www.curseforge.com/minecraft/mc-mods/mekanism-generators)
[Ars Creo (by baileyholl2)](https://www.curseforge.com/minecraft/mc-mods/ars-creo)
[Botania (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/botania)
[Domum Ornamentum (by OrionOnline)](https://www.curseforge.com/minecraft/mc-mods/domum-ornamentum)
[Simple Voice Chat (by henkelmax)](https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat)
[Basic Nether Ores [Forge/Fabric] (by cScotPlay)](https://www.curseforge.com/minecraft/mc-mods/basic-nether-ores)
[NetherPortalFix (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/netherportalfix)
[Sophisticated Core (by P3pp3rF1y)](https://www.curseforge.com/minecraft/mc-mods/sophisticated-core)
[Pipez (by henkelmax)](https://www.curseforge.com/minecraft/mc-mods/pipez)
[Mystical Agriculture (by BlakeBr0)](https://www.curseforge.com/minecraft/mc-mods/mystical-agriculture)
[Map Tooltip (by VendoAU)](https://www.curseforge.com/minecraft/mc-mods/map-tooltip)
[Redstone Pen (by wilechaote)](https://www.curseforge.com/minecraft/mc-mods/redstone-pen)
[Meet Your Fight (by Lykrast)](https://www.curseforge.com/minecraft/mc-mods/meet-your-fight)
[Biomes O' Plenty (by Forstride)](https://www.curseforge.com/minecraft/mc-mods/biomes-o-plenty)
[Crafting Tweaks (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/crafting-tweaks)
[Elytra Slot (Forge) (by TheIllusiveC4)](https://www.curseforge.com/minecraft/mc-mods/elytra-slot)
[Balm (Forge Edition) (by BlayTheNinth)](https://www.curseforge.com/minecraft/mc-mods/balm)
[Sophisticated Backpacks (by P3pp3rF1y)](https://www.curseforge.com/minecraft/mc-mods/sophisticated-backpacks)
[Aquaculture 2 (by Shadowclaimer)](https://www.curseforge.com/minecraft/mc-mods/aquaculture)
[AppleSkin (by squeek502)](https://www.curseforge.com/minecraft/mc-mods/appleskin)
[GeckoLib (by ThanosGecko)](https://www.curseforge.com/minecraft/mc-mods/geckolib)
[CodeChicken Lib 1.8.+ (by covers1624)](https://www.curseforge.com/minecraft/mc-mods/codechicken-lib-1-8)
[Construction Wand (by ThetaDev)](https://www.curseforge.com/minecraft/mc-mods/construction-wand)
[TerraBlender (Forge) (by TheAdubbz)](https://www.curseforge.com/minecraft/mc-mods/terrablender)
[ConnectedTexturesMod (by tterrag1098)](https://www.curseforge.com/minecraft/mc-mods/ctm)
[FastWorkbench (by Shadows_of_Fire)](https://www.curseforge.com/minecraft/mc-mods/fastworkbench)
[YDM's Weapon Master (by YourDailyModderx)](https://www.curseforge.com/minecraft/mc-mods/ydms-weapon-master)
[Fire Extinguisher - Stop Fire (by Kaworru)](https://www.curseforge.com/minecraft/mc-mods/fire-extinguisher)
[Equipment Compare [Forge] (by Grend_G)](https://www.curseforge.com/minecraft/mc-mods/equipment-compare)
[Patchouli (by Vazkii)](https://www.curseforge.com/minecraft/mc-mods/patchouli)
[Rubidium (by Asek3)](https://www.curseforge.com/minecraft/mc-mods/rubidium)
[Scuba Gear (by ModdingLegacy)](https://www.curseforge.com/minecraft/mc-mods/scuba-gear)
[Valhelsia Core (by ValhelsiaTeam)](https://www.curseforge.com/minecraft/mc-mods/valhelsia-core)

### Tried reproducing with just Tinkers?

No

### Performance Enchancers

Magnesium, Other (specify under other mods)

### Searched for known issues?

Checked pinned issues

**Comment by Birbolion** [comment]
Forgot to specify, but Rubidium is the "other" performance enhancer.

**Comment by ShinyPorygon** [comment]
If you're going to leave a long wall of log text either put it on pastebin or upload the whole log file as a text file. A long wall of text makes the issue hard to follow

**Comment by KnightMiner** [comment]
Fixed in 3.6.2.92
